src/openrct2/paint/tile_element/Paint.Path.cpp

```diff
@@ -900,14 +900,16 @@ static void PaintPatrolAreas(paint_session& session, const PathElement& pathEl)
 {
     if (gStaffDrawPatrolAreas != 0xFFFF)
     {
+        // TODO: Split this into two.
```

Comment:
two what?

Reply:
Its gStaffDrawPatrolAreas holding two things, had has the entity id in 0x7FFF and one bit 0x8000 is used to determine if its the staff list or not, so its best to split them.

Reply:
See #16651